server/services/gameList.js

```diff
@@ -74,7 +98,7 @@ module.exports = class GameListService {
         .select({
             'settings.general.name': 1,
             'settings.general.playerLimit': 1,
-            state: 1
+            state: $filter
```

Comment:
What's this for?

Reply:
Oh wait, that was definitely _not_ supposed to be there. Looks like I accidentally commited it after trying something out.